src/browser_action/on-each-interaction.js

```diff
@@ -11,43 +11,139 @@
  limitations under the License.
 */
 
+
```

Comment:
@philipwalton when you get a chance could you review this code? It's basically a massively stripped down version of your web-vitals.js code to allow [each interaction to continue to give similar output to the INP debug info](https://web.dev/articles/debug-cwvs-with-web-vitals-extension#inp_debug_information).